_stbt/transition.py

```diff
@@ -222,7 +222,7 @@ def strict_diff(f1, f2, region, mask_image):
     if mask_image is not None:
         absdiff = cv2.bitwise_and(absdiff, mask_image, absdiff)
 
-    return absdiff.any()
+    return numpy.count_nonzero(absdiff) > 50 or (absdiff > 20).any()
```

Comment:
These are fixed numbers independent of the size of frame passed in.  They may make sense for 720p (~3,000,000 elements) but would make less sense if we've passed in a region or applied a mask. OTOH 50 elements is a 4x4px region which perhaps is small enough for any image.